ui/app/utils/version.ts

```diff
@@ -5,4 +5,4 @@
  * It serves as the single source of truth for version information.
  */
 
-export const VERSION = "2025.6.3";
+export declare const __APP_VERSION__: string;
```

Comment:
Can this file be deleted now since we declare `__APP_VERSION__` in globals.d.ts?

Reply:
yes